app/commands/decidim/vocdoni/admin/setup_election.rb

```diff
@@ -21,6 +21,7 @@ def call
           transaction do
             log_action
             update_election
```

Comment:
Same here, as we're changing the status but the job could fail, we should move the `update_election` logic to the job and also change the flash message

Reply:
In this case I'd would leave it as it is.
There's a safe guard to create the election in case failed in Vocdoni.
This process allows a loading spinner to wait for the vocdoni creation

Reply:
Ok, if there's a safeguard then let's leave it as it is